A context manager to suppress unwanted prints (#1483)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where **openapi-codegen** entered into an infinite loop on circular references in the swagger file.
* The **format** command will now set `fromVersion: 6.2.0` for widgets with 'metrics' data type.
* Updated the **find-dependencies** command to support generic modules, definitions, fields and types.

> * Added a function which gives the ability to temporarily suppress console output.

Reviewer: ```suggestion
* Added a tool that gives the ability to temporarily suppress console output.
```

Author: Done
